src/libecalc/presentation/yaml/yaml_types/components/yaml_pump.py

```diff
@@ -30,6 +43,11 @@ class Config:
 
     energy_usage_model: YamlTemporalModel[str]
 
+    # TODO: Same for compressor and pump and ...?
+    stream_conditions: Optional[
```

Comment:
Should be handled in a separate class from the one used in system. The schema will now say that this can be specified when used in a system, which is confusing.